### ISSUE TITLE
Any interest in a useful Pandoc preprocessor/wrapper scripts collection?

### ISSUE THREAD

**Issue by paultopia** [kotfu/marked-bonus-pack#11]
Hi there, 

I just hacked together a [quick preprocessor script](https://github.com/paultopia/marked-citation-preprocessor) in Python to solve a bunch of common problems with paths when using pandoc as a processor with pandoc-citeproc.  It occurs to me that others might have useful preprocessor scripts, wrapper scripts for Pandoc, etc. etc.  Judging by the Marked 2 forums, there are a million people hacking around to get Pandoc to work right as a processor.  

Would there be any interest in including such scripts as a separate category in this collection?

**Comment by kotfu** [comment]
Absolutely. I don't use pandoc so I'm not much help with the specifics, but I'm happy to collect useful scripts and associated documentation. Do you want to create a pull request or should I take your script and readme and commit them as the starting point?

**Comment by paultopia** [comment]
Sure, I can do a PR!  May take me a bit tho, as I'm moving this week.
